src/backend/api/meals.js

```diff
@@ -5,11 +5,119 @@ const knex = require("../database");
 router.get("/", async (request, response) => {
   try {
     // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    //const titles = await knex("meal_application").select("title");
+    //response.json(titles);
+    await knex("meals").insert({
+      id: "12",
+      title: "Egg fried rice",
+      discription: "Egg with rice and some chicken",
+      location: "Glostrup",
+      when_: new Date(),
+      max_reservations: 10,
+      price: 250,
+      created_date: new Date(),
+    });
   } catch (error) {
     throw error;
   }
 });
+router.get("/", async (request, response) => {
+  try {
+    if ("maxPrice" in request.query) {
+      const maxPrice = request.query.maxPrice;
```

Comment:
```suggestion
      const maxPrice = Number(request.query.maxPrice);
```
i think you have to be careful when you get a string query parameter and make sure it's a number